sourced/ml/mining/cmd/clickhouse2deps.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I bet that YAML will be more user-friendly here.

Reply:
done